packages/gatsby-theme-docs/src/layouts/internals/layout-release-note.js

```diff
@@ -5,22 +5,22 @@ import styled from '@emotion/styled';
 import { css } from '@emotion/core';
 import SpacingsStack from '@commercetools-uikit/spacings-stack';
 import SpacingsInline from '@commercetools-uikit/spacings-inline';
+import Stamp from '@commercetools-uikit/stamp';
 import { Markdown, designSystem } from '@commercetools-docs/ui-kit';
 
 const ReleaseNoteTitle = Markdown.withAnchorLink(Markdown.H3);
 const DateElement = styled.div`
-  line-height: ${designSystem.typography.lineHeights.releaseNoteDate};
+  line-height: ${designSystem.typography.lineHeights.small};
 `;
+// Wraps the uikit `Stamp` component to allow custom color schemes.
+// The `> div` selector should target the `Stamp` container element.
 const ReleaseNoteType = styled.div`
-  padding: ${designSystem.dimensions.spacings.xs}
-    ${designSystem.dimensions.spacings.s};
-  font-size: ${designSystem.typography.fontSizes.extraSmall};
-  line-height: ${designSystem.typography.lineHeights.releaseNoteDate};
-  border: 1px solid transparent;
-  border-radius: ${designSystem.tokens.borderRadiusForReleaseNoteType};
-  text-transform: capitalize;
-
-  ${getTypeStyles}
+  > div {
+    color: ${designSystem.colors.light.textPrimary};
+    line-height: ${designSystem.typography.lineHeights.small};
+    font-size: ${designSystem.typography.fontSizes.extraSmall};
+    ${getTypeStyles}
+  }
```

Comment:
This is basically how we override the styles of the `Stamp` component.
Not ideal, we need to improve the `Stamp` component a bit in order to avoid this.